Main/Source/command.cpp

```diff
@@ -503,13 +503,13 @@ truth commandsystem::SetItemLabel(character* Char)
   {
     itemvector ToAddLabel;
     game::DrawEverythingNoBlit();
-    Char->GetStack()->DrawContents(ToAddLabel, Char, CONST_S("What do you want to add a label?"), REMEMBER_SELECTED);
+    Char->GetStack()->DrawContents(ToAddLabel, Char, CONST_S("What do you want to inscribe?"), REMEMBER_SELECTED);
 
     if(ToAddLabel.empty())
       break;
 
     festring What = ToAddLabel[0]->GetLabel();
-    if(game::StringQuestion(What, CONST_S("How you want to label it?"), WHITE, 0, 20, true) == NORMAL_EXIT)
+    if(game::StringQuestion(What, CONST_S("What would you like to inscribe in this item?"), WHITE, 0, 20, true) == NORMAL_EXIT)
```

Comment:
inscribe ON this item :)

Reply:
thx! xD

Reply:
added some more on(s), not sure if all correct tho